modules/wordads/wordads.php

```diff
@@ -92,6 +92,13 @@ function init() {
 		}
 
 		$this->insert_adcode();
+
+		if ( '/ads.txt' === $_SERVER['REQUEST_URI'] ) {
+			header( 'Content-Type: text/plain; charset=utf-8' );
+			$ads_txt_content = apply_filters( 'wordads_ads_txt', WordAds_API::get_wordads_ads_txt() );
```

Comment:
Can you add some docs here?

[example](https://github.com/Automattic/jetpack/blob/branch-6.0/class.photon.php#L476-L491)